configure

```diff
@@ -4352,10 +4352,10 @@ fi
 
 if enabled cuda_nvcc; then
     nvcc_default="nvcc"
-    nvccflags_default="-gencode arch=compute_30,code=sm_30 -O2"
+    nvccflags_default="-gencode arch=compute_52,code=sm_52 -O2"
```

Comment:
I don't think anybody in the wider livepeer transcoder community uses Kepler based GPUs, but if they do will they stop working with this change?

If yes, we should mention that in the go-livepeer GPU docs https://github.com/livepeer/go-livepeer/blob/master/doc/gpu.md in the same PR that will pick up this version of FFmpeg.

Reply:
That's right, it won't work on compute capability 30. Current `configure.sh` doesn't work on CUDA 11.4 for me, and shouldn't work starting from CUDA 11.0, because compute_30 support is dropped from there, according to Nvidia docs.

Reply:
Makes sense thx!